client/app/cloud/project/compute/infrastructure/list/cloud-project-compute-infrastructure-list.html

```diff
@@ -3,89 +3,64 @@
-                        </button>
-                    </li>
-                    <li class="divider" role="presentation"></li>
-                    <li>
-                        <button type="button" class="btn fs14"
-                                at-internet-click="{ name : 'cloud-project::cloud-project-compute::cloud-project-compute-infrastructure-buy-ip' }"
-                                data-ng-bind=":: 'cpci_buy_ip_failover' | translate"
-                                data-ng-click="ComputeInfrastructureListCtrl.InfrastructureService.buyIpFailOver()">
-                        </button>
-                    </li>
-                    <li class="divider"
-                        data-ng-if="ComputeInfrastructureListCtrl.TARGET === 'US'">
-                    </li>
-                    <li data-ng-if="ComputeInfrastructureListCtrl.TARGET === 'US'">
-                        <button type="button"
-                                class="btn fs14"
-                                data-ng-click="ComputeInfrastructureListCtrl.InfrastructureService.orderCredit()"
-                                data-translate="cpb_vouchers_add_credit_title">
-                        </button>
-                    </li>
-                    <li class="divider" role="presentation"></li>
-                    <li role="menuitem">
-                        <button type="button" class="btn fs14"
-                                at-internet-click="{ name : 'cloud-project::cloud-project-compute::cloud-project-compute-infrastructure-delete-project' }"
-                                data-ng-bind=":: 'cpb_details_project_delete' | translate"
-                                data-ng-click="ComputeInfrastructureListCtrl.InfrastructureService.openDeleteProjectModal()"
-                                write-right-required>
-                        </button>
-                    </li>
-                </ul>
-            </div>
-        </div>
-        <div class="col-xs-9 text-right">
-            <div class="d-inline-block">
-                <span class="oui-label align-middle d-inline-block mb-0 mr-3"
-                      data-translate="cpci_menu_view_label"></span>
-                <nav class="oui-button-group align-middle mb-0">
-                    <a class="oui-button oui-button_secondary"
-                       ui-sref="iaas.pci-project.compute.infrastructure.diagram()"
-                       data-translate="cpci_menu_view_diagram"></a>
-                    <a class="oui-button oui-button_primary"
-                       ui-sref="iaas.pci-project.compute.infrastructure.list()"
-                       data-translate="cpci_menu_view_list"></a>
-                </nav>
-            </div>
+    <div class="row">
+        <div class="col-xs-12 mb-5 text-right">
+            <span class="oui-label align-middle d-inline-block mb-0 mr-3"
+                  data-translate="cpci_menu_view_label"></span>
+            <nav class="oui-button-group align-middle mb-0">
```

Comment:
I don't think `nav` make sense here. `nav` is supposed to be used for main navigation sections :)

Reply:
Have changed this to `<div>`